src/bioregistry/data/bioregistry.json

```diff
@@ -73514,7 +73512,7 @@
       "version": "4.3",
       "version.iri": "https://www.orphadata.com/data/ontologies/ordo/last_version/ORDO_en_4.3.owl"
     },
-    "pattern": "^C?\\d+$",
```

Comment:
With this change, you wont validate any "normal" orpha codes, they don't have that "C"

Reply:
Yes but like I noted, it make more practical sense to push them together instead of having too, else how will you be able to say that both have the same uri_prefix?

Reply:
So far I have not heard a clear argument why the separation is necessary. I have good reasons why it is not _necessary_. I can only think of a single reason, which is being able to redirect xrefs from browsers to separate destinations depending on whether they are properties or diseases. Can you point me to the issue that clearly explains that this is what you are trying to solve?